edgar/financials.py

```diff
@@ -251,13 +251,18 @@ def _get_statement_meta_data(rows):
                     title_repeat = 0 if 'colspan' not in info.attrs or int(info.attrs['colspan']) == 1 else int(info.attrs['colspan']) - 1
                     # first th with tl class has title and unit specification
                     info_list = info.find('div').get_text('|', strip=True).split('|')
+
                     # e.g. shares in Thousands, $ in Millions
-                    unit_text = info_list[1]
+                    unit_text = ''
```

Comment:
@cdbattags just revisiting this PR...this one i'm not sure about. if you can find a case that doesn't have length 2 then i'd like to see it, otherwise i would actually prefer the runtime error here so that we're alerted to such cases.

Reply:
DBX 2019 (don't remember which statement), it doesn't provide a unit, hence number are good as they are.